src/amo/components/CollectionManager/index.js

```diff
@@ -115,6 +121,7 @@ export class CollectionManagerBase extends React.Component<Props, State> {
 
   componentWillUnmount() {
     if (this.timeout) {
+      invariant(this.props.clearTimeout, 'props.clearTimeout is undefined');
```

Comment:
I see that sometimes you add parentheses, and sometimes not. I think we should keep parentheses to indicate functions, but I'd get rid of the `prop.` prefix because when we get the invariant error, the file and line are indicated.